MediaWikiKit/MediaWikiKit/MWKSearchResult.m

```diff
@@ -20,10 +20,10 @@ + (NSValueTransformer*)wikidataDescriptionJSONTransformer {
 
 + (NSDictionary*)JSONKeyPathsByPropertyKey {
     return @{
-               @"displayTitle": @"title",
-               @"articleID": @"pageid",
-               @"thumbnailURL": @"thumbnail.source",
-               @"wikidataDescription": @"terms.description"
+               WMF_SAFE_KEYPATH(MWKSearchResult.new, displayTitle): @"title",
```

Comment:
reiterating: `WMF_SAFE_KEYPATH` (like the [libextobjc macro I borrowed from](https://github.com/jspahrsummers/libextobjc/blob/45babac1a457b0f59404a366f26bac2b5b5e82b2/extobjc/EXTKeyPathCoding.h#L38)) doesn't actually run the code in the first argument (i.e. no objects are created).  this happens because the compiler can determine the code is unreachable, and optimize it away.